src/components/interfaces/MOBILE_API.xml

```diff
@@ -1828,60 +1828,134 @@
     
     <struct name="GPSData">
         <description>Struct with the GPS data.</description>
-        <param name="longitudeDegrees" type="Float" minvalue="-180" maxvalue="180" mandatory="true">
-        </param>
-        <param name="latitudeDegrees" type="Float" minvalue="-90" maxvalue="90" mandatory="true">
-        </param>
-        <param name="utcYear" type="Integer" minvalue="2010" maxvalue="2100" mandatory="true">
+        <param name="longitudeDegrees" type="Float" minvalue="-180" maxvalue="180" since="5.0">
```

Comment:
suggest to keep longitudeDegrees unchanged

Reply:
@yang1070 This change is specified in proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0175-Updating-DOP-value-range-for-GPS-notification.md
Do you want us to miss this part?